run_project_tests.py

```diff
@@ -122,7 +122,7 @@ def get_relative_files_list_from_dir(fromdir):
 
 def platform_fix_name(fname, compiler, env):
     # canonicalize compiler
-    if compiler in {'clang-cl', 'intel-cl'}:
+    if compiler in {'clang-cl'}:
```

Comment:
It was a mistake to remove 'intel-cl' here; I didn't realize it was unrelated. I added it back in a later commit in this PR.